hcn/hcnpolicy.go

```diff
@@ -64,14 +64,18 @@ type SubnetPolicy struct {
 	Settings json.RawMessage  `json:",omitempty"`
 }
 
+// NatFlags are flags for portmappings.
+type NatFlags uint32
```

Comment:
Should this be enum? What are the flags you can pass? At the very least please just document some of them?

Reply:
I'm just trying to stay consistent with how we define flags elsewhere in ./hcn. I'll add variables for the two flags.

Reply:
Np